paparazzi-gradle-plugin/src/test/projects/verify-data-binding/src/main/res/layout/layout.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Have you tried using the <data/> tag?

Reply:
Not yet but I can add a test for that next

Reply:
I think let's not support `<data/>` tags; I feel like it's a Pandora's box and not worth the squeeze.

The original issue in https://github.com/cashapp/paparazzi/issues/1016 doesn't use it, and Paparazzi takes a fairly opinionated stance about separating business logic from view.  Instead, users can grab a handle to textview and set their logic-driven properties directly in test code.